README.md

```diff
@@ -38,55 +88,59 @@ By default, this package will look for your Apple Search Ads data in the `apple_
-Please create issues or open PRs against `master`. See [the Discourse post](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) for information on how to contribute to a package.
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_apple_search_ads/blob/main/dbt_project.yml) variable declarations to see the expected names.
 
-## Database Support
+```yml
+vars:
+    apple_search_ads_<default_source_table_name>_identifier: your_table_name 
+```
 
-This package has been tested on BigQuery, Snowflake, Redshift, PostgreSQL, and Databricks.
+## (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™    
+Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
 
-### Databricks Dispatch Configuration
-dbt `v0.20.0` introduced a new project-level dispatch configuration that enables an "override" setting for all dispatched macros. If you are using a Databricks destination with this package, you will need to add the dispatch configuration below (or a variation of that configuration) within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
+
+# 🔍 Does this package have dependencies?
+This dbt package is dependent on the following dbt packages. Please be aware that these dependencies are installed by default within this package. For more information on the following packages, refer to the [dbt hub](https://hub.getdbt.com/) site.
+> IMPORTANT: If you have any of these dependent packages in your own `packages.yml` file, we highly recommend that you remove them from your root `packages.yml` to avoid package version conflicts.
+    
 ```yml
-# dbt_project.yml
+packages:
+    - package: fivetran/apple_search_ads_source
+      version: [">=0.1.0", "<0.2.0"]
 
-dispatch:
-  - macro_namespace: dbt_utils
-    search_order: ['spark_utils', 'dbt_utils']
+    - package: fivetran/fivetran_utils
+      version: [">=0.3.0", "<0.4.0"]
+
+    - package: dbt-labs/dbt_utils
+      version: [">=0.8.0", "<0.9.0"]
 ```
+# 🙌 How is this package maintained and can I contribute?
+## Package Maintenance
+The Fivetran team maintaining this package _only_ maintains the latest version of the package. We highly recommend you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/apple_search_ads/latest/) of the package and refer to the [CHANGELOG](https://github.com/fivetran/dbt_apple_search_ads/blob/main/CHANGELOG.md) and release notes for more information on changes across versions.
+
```

Comment:
There was no need for a DECISIONLOG.md in this package, right?

Reply:
Can confirm -- no `DECISIONLOG` needed in this package.